doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -803,11 +803,11 @@ Per-worker relations expose the same data as their global counterparts, but have
 The `mz_active_peeks` view describes all read queries ("peeks") that are pending in the [dataflow] layer.
 
 <!-- RELATION_SPEC mz_internal.mz_active_peeks -->
-| Field       | Type               | Meaning                                                                                                           |
-| ----------- | ------------------ | --------                                                                                                          |
-| `id`        | [`uuid`]           | The ID of the peek request.                                                                                       |
-| `index_id`  | [`text`]           | The ID of the index the peek is targeting. Corresponds to [`mz_catalog.mz_indexes.id`](../mz_catalog#mz_indexes). |
-| `time`      | [`mz_timestamp`]   | The timestamp the peek has requested.                                                                             |
+| Field       | Type               | Meaning                                                                                                                                              |
+| ----------- | ------------------ |------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `id`        | [`uuid`]           | The ID of the peek request.                                                                                                                          |
+| `index_id`  | [`text`]           | The ID of the collection the peek is targeting. When targeting an index, this corresponds to [`mz_catalog.mz_indexes.id`](../mz_catalog#mz_indexes). |
```

Comment:
```suggestion
| `index_id`  | [`text`]           | The ID of the collection the peek is targeting. Corresponds to [`mz_catalog.mz_indexes.id`](../mz_catalog#mz_indexes), [`mz_catalog.mz_materialized_views.id`](../mz_catalog#mz_materialized_views), [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources), or [`mz_catalog.mz_tables.id`](../mz_catalog#mz_tables). |
```

This is how we handle this when documenting other relations (I copied this from `mz_compute_dependencies`).